lib/probes/morgan.js

```diff
@@ -20,7 +20,7 @@ module.exports = function (morgan) {
   const pmorgan = new Proxy(morgan, {
     // call target/morgan as a function
     apply (target, thisArg, argumentsList) {
-      if (ao.cfg.insertTraceIdsIntoMorgan) {
+      if (ao.cfg.insertTraceIdsIntoLogs) {
         argumentsList = tryInsertion(morgan, argumentsList)
       } else if (ao.cfg.createTraceIdsToken === 'morgan') {
```

Comment:
in lib/index.js
```
// if inserting into morgan then a token must be created.
if (config.insertTraceIdsIntoLogs) {
  config.createTraceIdsToken = 'morgan'
}
```
is there somewhere else where `createTraceIdsToken` can be assigned?
why is the above code in index.js and not in this file?

I'm also wondering why create a token when it is not immediately used and the function that uses it creates it anyways

is `token` used somewhere else?

Reply:
This is removed in next PR which will be refactoring `morgan`....